webapp/locales/de.json

```diff
@@ -1,736 +1,2246 @@
 {
-  "components": {
-    "password-reset": {
-      "request": {
-        "form": {
-          "description": "Eine Mail zum Zurücksetzen des Passworts wird an die angegebene E-Mail Adresse geschickt.",
-          "submit": "Email anfordern",
-          "submitted": "Eine E-Mail mit weiteren Instruktionen wurde verschickt an <b>{email}</b>"
+    "login": {
+        "copy": {
+            "translation": "Falls Du bereits ein Konto bei Human Connection hast, melde Dich bitte hier an.",
```

Comment:
Why do we have those keys "translation" and "notes" all over the place? @Mogge misconfiguration?

Reply:
See #2305. I changed the export to
![grafik](https://user-images.githubusercontent.com/15882241/69237752-aa658000-0b96-11ea-90f4-dc85c5efaac4.png)

I close this PR